WizardWorldApi/WizardWorld.Application/Requests/MagicalCreatures/MagicalCreatureDto.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using WizardWorld.Persistance.Models.MagicalCreatures;
+
+namespace WizardWorld.Application.Requests.MagicalCreatures {
+    public class MagicalCreatureDto {
+        public Guid Id { get; set; }
+        public string Name { get; set; }
+        public string Description { get; set; }
+        public CreatureClassificationByMinistryOfMagic Classification { get; set; }
+        public CreatureStatus Status { get; set; }
+        public CreatureDangerLevel DangerLevel { get; set; }
+        public ICollection<CreatureRelation> CreatureRelations { get; set; }
```

Comment:
This should be `ICollection<string>`

Reply:
Why? CreatureRelation is a type

Reply:
Because we want to only return a list of strings to the user.